lib/go-tc/deliveryservices.go

```diff
@@ -172,11 +174,12 @@ type DeliveryServiceV11 struct {
 
 type DeliveryServiceNullableV30 struct {
 	DeliveryServiceNullableV15
-	Topology           *string `json:"topology" db:"topology"`
-	FirstHeaderRewrite *string `json:"firstHeaderRewrite" db:"first_header_rewrite"`
-	InnerHeaderRewrite *string `json:"innerHeaderRewrite" db:"inner_header_rewrite"`
-	LastHeaderRewrite  *string `json:"lastHeaderRewrite" db:"last_header_rewrite"`
-	ServiceCategory    *string `json:"serviceCategory" db:"service_category"`
+	Topology             *string `json:"topology" db:"topology"`
+	FirstHeaderRewrite   *string `json:"firstHeaderRewrite" db:"first_header_rewrite"`
+	InnerHeaderRewrite   *string `json:"innerHeaderRewrite" db:"inner_header_rewrite"`
+	LastHeaderRewrite    *string `json:"lastHeaderRewrite" db:"last_header_rewrite"`
+	ServiceCategory      *string `json:"serviceCategory" db:"service_category"`
+	MaxRequestHeaderSize *int    `json:"maxRequestHeaderSize" db:"max_request_header_size"`
```

Comment:
Since 5.0 is leaving the station, we probably need a new `DeliveryServiceNullableV31` struct and all the other changes that come with that, since this will likely be 3.1. Right @ocket8888? This shouldn't be backported.

Reply:
Well hopefully the next release can include the DS `active` flag change, so we may actually want to go to 4.0. That'll be a breaking change that needs a new major version bump.

Reply:
The DS `active` flag change(that I worked on) is not modifying the structure. It just lets us query by the `active` flag.

Reply:
He's referring to https://github.com/apache/trafficcontrol/blob/master/blueprints/ds-active.md which hasn't been started yet. That blueprint would call for a major version bump.

_This_ PR could add it as 3.1, then if we get to ☝️ before 5.1 is released, that API could still be added as 4.0. Then 3.1 == 4.0 like how 1.5 == 2.0 (if that makes sense). It _would_ be cleaner if we were able to have just one or the other, but I don't think it's a big deal if we end up having to add both.